recipe/meta.yaml

```
@@ -1,11 +1,9 @@
 {% set gcc_constraint = ">=6,<15.0a0" %}
```

GCC 15 support was added in CUDA 13. https://docs.nvidia.com/cuda/cuda-toolkit-release-notes/index.html

> New host compiler support in CUDA 13.0:
> - Clang 20
> - GCC 15

```suggestion
{% set gcc_constraint = ">=6,<16.0a0" %}
```